packages/core/package.json

```diff
@@ -61,6 +61,7 @@
     "ajv": "~6.12.0",
     "chalk": "~4.1.0",
     "commander": "~6.2.0",
+    "cosmiconfig": "^7.0.0",
```

Comment:
I'd rather prefer "~" instead of "^". We have been trying to eliminate all major versions in packages for long time now ._.